apps/cave/hooks/usePermit/usePermit.ts

```diff
@@ -36,7 +37,7 @@ const signPermit = async (
  */
 const PERMITTABLE_TOKENS = [...Object.values(CNV_ADDRESS), DAI_ADDRESS[chain.mainnet.id]]
 const isTokenPermissible = (token: Token) => {
-  return !!PERMITTABLE_TOKENS.includes(token?.address)
+  return !!PERMITTABLE_TOKENS.includes(token?.address) && token.chainId !== chain.localhost.id
```

Comment:
do we need this check? It's already checked in line 51

Reply:
👍